redi-connect-front/src/pages/front/login/Login.tsx

```diff
@@ -70,144 +49,142 @@ export default function Login() {
     try {
       const accessToken = await login(formValues.username, formValues.password);
       saveAccessToken(accessToken);
-      const myProfile = await fetchSaveRedProfile(accessToken);
-      history.push("/app/dashboard");
+      await fetchSaveRedProfile(accessToken);
+      history.push('/app/dashboard');
     } catch (err) {
-      setLoginError("Invalid username or password");
+      setLoginError('You entered an incorrect email, password, or both.');
     }
     actions.setSubmitting(false);
   };
 
-  return (
-    <LoggedOutLayout>
-      <Formik
-        initialValues={initialValues}
-        validationSchema={validationSchema}
-        onSubmit={submitForm}
-        render={props => <Form {...props} loginError={loginError} />}
-      />
-    </LoggedOutLayout>
-  );
-}
-
-const Form = withStyles(styles)(
-  (
-    props: FormikProps<LoginFormValues> & {
-      loginError: string;
-      classes: {
-        loginError: string;
-        headingMargin: string;
-        navigationBox: string;
-      };
-    }
+  const change = (
+    name: any,
+    handleChange: any,
+    setFieldTouched: any,
+    e: any
   ) => {
-    const {
-      classes,
-      loginError,
-      values: { username, password },
-      errors,
-      touched,
-      handleChange,
-      isValid,
-      isSubmitting,
-      setFieldTouched,
-      submitForm,
-    } = props;
-
-    const change = (name: any, e: any) => {
-      e.persist();
-      handleChange(e);
-      setFieldTouched(name, true, false);
-    };
+    e.persist();
+    handleChange(e);
+    setFieldTouched(name, true, false);
+  };
 
-    return (
-      <Container maxWidth="sm">
-        <Typography
-          variant="h5"
-          align="center"
-          className={classes.headingMargin}
+  return (
+    <LoggedOutLayout modifier="page--color-half">
+      <Columns vCentered>
```

Comment:
> I played around with `Columns` a bit. Removing `vCentered` seems to have no effect. If it doesn't serve a purpose, remove it?

It does have an effect of cantering the right column based on the hight of the left one. Remove the rendered class in the browser and it should be visible.

Reply:
> I'm having slightly cold feet about `react-bulma-components`. I did a deep dive and found there's been a few proper projects trying to wrap bulma into React components:
> 
> ![Screenshot 2020-03-31 at 20 57 26](https://user-images.githubusercontent.com/10571311/78064584-4481d400-7392-11ea-8010-e8d0e2cbf172.png)
> 
> `react-bulma-components` is the leader of the pack, so I did some more digging:
> 
> * there's been an issue hanging around for about 6 months about writing up TypeScript types - no-body wants to do it
> * there are some ok-ish propTypes documentation - but after lots of fiddling I still can't access it via intellisense/autocomplete in vscode (do you have any tips re. this?)
> * the docs aren't really good and progress is slow on getting that up. It's a real pity, since some of the sister projects (notably [bloomer](https://bloomer.js.org/) and [rbx](https://dfee.github.io/rbx/)) have done a much better job ... but they've stagnated
> 
> All of this to say: `react-bulma-components` is "good enough" for now. Let's cross our fingers and hope for the best. Though I wouldn't be surprised if we'd have to leave it behind in a year or two.
> 
> actually, it might be a fun opensource project to contribute to.... hmmm

Yup, I took a look at almost all the others you mentioned, did not compare with the fancy graph but I investigated a bit. At the end, currently there is no better then the one we are using. I dug into the source a bit and it build pretty good. Maybe in the future we can think of supporting the project..